tests/dummy/app/routes/index.js

```diff
@@ -106,8 +106,8 @@ export default Ember.Route.extend({
     model.filteredColumnDefs[3].filter = { // alignment
       type: 'select',
       selectOptions: [
-        { value: 'Good' },
-        { value: 'Evil' },
+        { value: 'Good', label: 'Positive' },
```

Comment:
This example is potentially confusing because someone looking at the example page won't know why the select options suddenly become "Positive" and "Negative" even though the data rows still say "Good" and "Evil." If we're going to change this in the dummy app/examples page, we should also add a little bit of text that explains why this is happening.